Cargo.lock

```diff
@@ -710,7 +716,7 @@ dependencies = [
  "alloy-rlp",
  "arbitrary",
  "derive_arbitrary",
- "derive_more",
+ "derive_more 0.99.18",
```

Comment:
This is duplicated until a new alloy release, so this is fine

Reply:
And revm also I think, just did a PR here: https://github.com/bluealloy/revm/pull/1713